scenarios/register.go

```diff
@@ -0,0 +1,7 @@
+package scenarios
+
+// RegisterScenarios registers all scenarios. It is intended to be called by Go
+// applications using omes as a library to run scenarios.
+func RegisterScenarios() {
+	// Scenarios are registered automatically on import of this package.
+}
```

Comment:
Rather than exposing this and needing the caller to call it, maybe it'd make more sense to do a `NewScenarioRunner` which accepts the options, calls this, and returns the runner?

Reply:
The scenarios are currently loaded automatically for the CLI caller by Go's package `init()` mechanism, so it's only library callers that need a way to load them. I did it this way because I thought that the authors of the library might not appreciate a redundant call being placed in a code path when loading was already handled by the `init` mechanism. WDYT?

Reply:
Also I think the import graph might be showing that this would go against the design: `scenarios` imports `loadgen`, so I don't think there's anywhere we can import `scenarios` in the CLI-start-up sequence.

Reply:
Bleh, Go imports. This is fine, it's no big deal. It seems a touch unnecessary, so if we could make it happen automatically that'd be great (I don't think having a redundant call matters, it's idempotent).